03_launch_mgmt_cluster.sh

```diff
@@ -261,7 +261,11 @@ function update_capm3_imports(){
   pushd "${CAPM3PATH}"
 
   # Modify the kustomization imports to use local BMO repo instead of Github Main
-  make kustomize
+  if [[ "${CAPM3BRANCH}" == "release-1.5" ]] || [[ "${CAPM3BRANCH}" == "release-1.4" ]]; then
```

Comment:
Should 1.6 be included as well, or did you backport?

Reply:
it was added few months back, release 1.6 has the phony